sample/src/main/java/com/davidecirillo/multichoicesample/sampleCustomView/SampleCustomActivity.java

```diff
@@ -33,33 +28,26 @@ protected int setActivityIdentifier() {
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
         setUpMultiChoiceRecyclerView();
     }
 
+    @Override
+    protected void onSaveInstanceState(Bundle outState) {
+        ((MultiChoiceAdapter) multiChoiceRecyclerView.getAdapter()).onSaveInstanceState(outState);
```

Comment:
This way don't need to have a field in the activity, so it's pointless, i know 100% that my adapter will be a MultiChoiceAdapter one. It's fine